Description: Fixes requirements.txt
 Without this patch, keystone wouldn't start.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-23

--- keystone-8.0.0~rc1.orig/requirements.txt
+++ keystone-8.0.0~rc1/requirements.txt
@@ -33,6 +33,6 @@ oslo.utils>=2.0.0 # Apache-2.0
 oauthlib>=0.6
 pysaml2>=2.4.0
 dogpile.cache>=0.5.4
-jsonschema!=2.5.0,<3.0.0,>=2.0.0
+jsonschema
 pycadf>=1.1.0
 msgpack-python>=0.4.0
